### DISCUSSION TITLE
モンスターボールを手に持つのではなく投げて発動するようにしてほしい

### DISCUSSION THREAD

**Discussion by HiddenNameZ** [hengband/hengband#1948] [Ideas]
今の仕様は持ち替えが結構めんどいし、耐性が抜けたりもするしであまり良いことがない気がします、騎乗職も移動先選べなくなるし
人形みたいにvコマンドで投げて捕まえたり出したりするようにできないでしょうか

**Comment by dis-** [comment]
ペットシステムは非常に強力なので、モンスターボールによる片手の占有をなくす場合にはなんらかのバランス調整を入れる必要があります。
単純にユーザビリティを向上させるだけなら「持ち物欄から直接発動できるようにするが、発動には片手の空きを要求する」くらいになります。
装備欄の占有をなくすのであれば、相応のデメリットを考える必要がありますね。
